src/main/java/cz/brmlab/yodaqa/io/web/WebAnswerPrinter.java

```diff
@@ -45,12 +45,11 @@ public void process(JCas jcas) throws AnalysisEngineProcessException {
 		while (answerit.hasNext()) {
 			Answer a = ((Answer) answerit.next());
 			QuestionAnswer qa = new QuestionAnswer(a.getText(), a.getConfidence());
-			if (a.getPassageIDs() != null) {
-				for (Integer PassageID : a.getPassageIDs().toArray()) {
-					qa.addToPassageList(PassageID);
+			if (a.getSnippetIDs() != null) { //AnsweringSnippet ID should NOT be NULL!!!!!!!!!!!!!!!!!!
+				for (Integer PassageID : a.getSnippetIDs().toArray()) {
```

Comment:
Style (I know it carried over from your older code): Let's be careful to distinguish type/class names (starts with capital letter) from variables, properties, methods (starts with lower-case letter).